common/lib/dependabot/shared_helpers.rb

```diff
@@ -191,6 +191,13 @@ def self.with_git_configured(credentials:)
       reset_global_git_config(backup_git_config_path)
     end
 
+    # Handle SCP-style git URIs
+    def self.scp_to_standard(uri)
+      return uri unless uri.start_with?("git@")
```

Comment:
We have seen reports of users using alternate user names here but I think this is a good start and we can see if anything comes up.

Reply:
Agreed, thanks for pointing that out!